hexa/pipelines/migrations/0051_pipelineversion_version_number_cleanup.py

```diff
@@ -0,0 +1,27 @@
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [("pipelines", "0050_pipelineversion_set_version_numbers")]
+
+    operations = [
+        migrations.AddConstraint(
+            model_name="pipelineversion",
+            constraint=models.UniqueConstraint(
+                fields=("pipeline", "version_number"),
+                name="unique_pipeline_version_number",
+            ),
+        ),
+        migrations.AddIndex(
+            model_name="pipelineversion",
+            index=models.Index(
+                fields=["pipeline", "version_number"],
+                name="index_pipeline_version_number",
+            ),
+        ),
+        migrations.AlterField(
+            model_name="pipelineversion",
+            name="name",
+            field=models.CharField(max_length=250, null=True, blank=True),
+        ),
```

Comment:
This migration is going to break if you revert it after pipeline versions have been created without names no ?

Reply:
Indeed, good catch thanks, it will likely break
I have not caught that during my testing, I will have a look and possibly provide a revert script filling the names

Reply:
Added 0052 to fill empty values